src/pygama/raw/orca/orca_SIS3316.py

```diff
@@ -0,0 +1,229 @@
+import logging
+import sys
+from typing import Any
+
+import numpy as np
+
+from pygama.raw.orca.orca_header import OrcaHeader
+from pygama.raw.orca.orca_packet import OrcaPacket
+from pygama.raw.raw_buffer import RawBufferLibrary
+
+from .orca_base import OrcaDecoder, get_ccc
+
+log = logging.getLogger(__name__)
+
+
+class ORSIS3316WaveformDecoder(OrcaDecoder):
+    """Decoder for FlashCam ADC data written by ORCA."""
+
+    def __init__(self, header: OrcaHeader = None, **kwargs) -> None:
+        # self.decoder_name = "ORSIS3316WaveformDecoder"
+        # self.orca_class_name = "ORSIS3316Model"
```

Comment:
Please remove all commented code, leave only comments documenting the code. Please fix this also below.